spk/vim/src/dsm-control.sh

```diff
@@ -13,10 +13,14 @@ case $1 in
         exit 0
         ;;
     status)
-        exit 1
+	if [ -e /usr/local/bin/vim ]; then
```

Comment:
Realistically, the package could be considered broken only if `/usr/local/vim/bin/vim-utf8` is not available. The symlink is just to make life easier, but it's hardly critical if it's not there.